aws/README.md

```diff
@@ -62,64 +71,208 @@ docker build -t aws \
 
 ```
-docker build -t aws \
---secret id=AWS_ACCESS_KEY_ID \
---secret id=AWS_SECRET_ACCESS_KEY \
---build-arg AWS_REGION=${AWS_REGION} \
---build-arg BINARIES_IMG=quay.io/confidential-containers/podvm-binaries-centos-amd64 \
---build-arg PODVM_DISTRO=centos \
--f Dockerfile .
+export PODVM_AMI_ID=$(aws ec2 describe-images --query "Images[*].[ImageId]" --filters "Name=name,Values=peer-pod-ami" --region ${AWS_REGION} --output text)
+echo ${PODVM_AMI_ID}
 ```
 
->Note: images for CentOS Stream haven't been built since the release 0.8.0 of this project.
+## Build CAA container image
 
-- Note down your newly created AMI_ID
+> **Note**: If you have made changes to the CAA code and you want to deploy those changes then follow [these instructions](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/install/README.md#building-custom-cloud-api-adaptor-image) to build the container image from the root of this repository.
 
-Once the image creation is complete, you can use the following CLI command as well to
-get the AMI_ID. The command assumes that you are using the default AMI name: `peer-pod-ami`
+If you would like to deploy the latest code from the default branch (`main`) of this repository then expose the following environment variable:
+
+```bash
+export registry="quay.io/confidential-containers"
+```
+
+## Deploy Kubernetes using EKS
+
+- Create cluster config file
+This config will create a single node cluster. Feel free to change it as per your requirement.
 
 ```
-aws ec2 describe-images --query "Images[*].[ImageId]" --filters "Name=name,Values=peer-pod-ami" --region ${AWS_REGION} --output text
+cat > ekscluster-config.yaml <<EOF
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: caa-eks
+  region: ${AWS_REGION}
+nodeGroups:
+  - name: ng-1-workers
+    labels: { role: workers }
+    instanceType: m5.xlarge
+    desiredCapacity: 1
+    privateNetworking: true
+    ssh:
+      allow: true # will use ~/.ssh/id_rsa.pub as the default ssh key
+EOF
 ```
+- Create cluster
+```
+eksctl create cluster -f ekscluster-config.yaml
+```
+
+Wait for the cluster to be created.
 
-### Option-2: Using precreated QCOW2 image
+- Install `fuse` package on the worker node
+This is required for `nydus snapshotter` that is used for CoCo.
+You can either SSH to the node, or run a debug shell and install the `fuse` package
 
-- Download QCOW2 image
+
+## Deploy CAA
+
+### Create the credentials file
+```
+cat <<EOF > install/overlays/aws/aws-cred.env
+AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+EOF
 ```
-mkdir -p qcow2-img && cd qcow2-img
 
-curl -LO https://raw.githubusercontent.com/confidential-containers/cloud-api-adaptor/staging/podvm/hack/download-image.sh
+### Update the `kustomization.yaml` file
+
+Run the following command to update the [`kustomization.yaml`](../install/overlays/aws/kustomization.yaml) file:
```

Comment:
Hey Pradipta - is this intended to be a point in time reference, or something longer lasting? If it's the later then I don't know whether it would be better to instruction them to edit the existing `install/overlays/aws/kustomization.yaml` file rather than overwrite it, so things like the CAA new are not fixed?

You might be able to do something with the caa provisioner to help with the config, but I understand if you don't want to get into that.

Reply:
Thanks for the suggestion Steve. It does make sense to instruct direct editing of the kustomization.yaml. However, currently for AWS we only need one variable to be provided and the rewriting helps to get started quickly. 
Going forward, I will explore `sed` commands to edit the values from CLI instead of re-writing. Hope this is ok..

Reply:
Sure, if it's a short-term hard-coded doc then that's fine - I just don't want AWS users to get stuck on the 0.8 release of CAA once we've updated to 0.9 based on main

Reply:
BTW, another option would be to modify CM after creation, can be done in 2 commands, not sure it's better though...